### PR TITLE
[RIO-2024]Changed Logo Even, Remove Organizers, Add New Sponsor

### PR DESCRIPTION
**Pull request by mauriciopgomes**
Changed Logo DevOps Days Rio
Remove Giovanni Martinss and Paloma Lataliza to Organization
Add New Sponsor (Grafana)

### CODE REVIEW

**Comment by nimbinatus** [comment]
Hi there! As this PR is changing the organizing team, you need to send that information to the info@ alias so global core can address the email lists and Slack per the [website directions](https://github.com/devopsdays/devopsdays-web/tree/main/utilities#local-organizers).

File: data/sponsors/grafana.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: "grafana"
+url: ""
```
Comment (phrawzty) [inline data/sponsors/grafana.yml:2]:
```suggestion
url: "https://grafana.com"
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

**Comment by mauriciopgomes** [comment]
> Hi there! As this PR is changing the organizing team, you need to send that information to the info@ alias so global core can address the email lists and Slack per the [website directions](https://github.com/devopsdays/devopsdays-web/tree/main/utilities#local-organizers).

sent a e-mail

**Comment by mauriciopgomes** [comment]
> Hi there! As this PR is changing the organizing team, you need to send that information to the info@ alias so global core can address the email lists and Slack per the [website directions](https://github.com/devopsdays/devopsdays-web/tree/main/utilities#local-organizers).

sent e-mail

**Pull request merged**